pyproject.toml

```diff
@@ -49,6 +49,7 @@ aiohttp = { version = "^3.9.3", optional = true }
 minijinja = { version = "^1.0.16", optional = true }
 google-generativeai = { version = "^0.7.0", optional = true }
 litellm = {version="^1.41.23", optional = true}
+logging = "^0.4.9.6"
```

Comment:
Logging is part of the standard python library, so we do not need to install it or have it as a dependency,